tests/src/nodes/nomos.rs

```diff
@@ -314,6 +314,10 @@ fn create_node_config(
         metrics: Default::default(),
         da: nomos_da::Settings {
             da_protocol: full_replication::Settings {
+                // Using private_key as a voter only for easy deployment.
+                // It doesn't mean that private_key will be used for anything like signing
+                // in the FullReplication.
+                voter: private_key,
```

Comment:
Can we use the public key instead?

Reply:
Good point. This can mislead. But, it's not trivial to derive a public key from this private key, because we're not assuming any key type for now. 

Actually, this argument name should be `id` (not `private_key`) because the caller of this function deals with `id`s (not `private_key`s): https://github.com/logos-co/nomos-node/blob/ffc4054baab826c489d14ac0f7e445692a6f3479/tests/src/nodes/nomos.rs#L236-L236.

I renamed the argument name: ffc4054.